src/SFML/Window/Unix/KeyboardImpl.cpp

```diff
@@ -279,172 +279,172 @@ std::map<std::string, sf::Keyboard::Scancode> getNameScancodeMap()
 {
     std::map<std::string, sf::Keyboard::Scancode> mapping;
```

Comment:
I believe unordered_map is preferred.
```suggestion
    std::unordered_map<std::string, sf::Keyboard::Scancode> mapping;
```